src/System Application/App/Email/src/Email/Outbox/EmailEditor.Codeunit.al

```diff
@@ -162,18 +162,22 @@ codeunit 8906 "Email Editor"
     var
         FileName: Text;
         Instream: InStream;
-        AttachmentName, ContentType : Text[250];
-        AttachamentSize: Integer;
     begin
         UploadIntoStream('', '', '', FileName, Instream);
-        if FileName = '' then
-            exit;
+        if FileName <> '' then
+            UploadAttachmentFromStream(EmailMessageImpl, FileName, Instream);
+    end;
 
+    internal procedure UploadAttachmentFromStream(EmailMessageImpl: Codeunit "Email Message Impl."; FileName: Text; Instream: InStream)
```

Comment:
Remove internal. No need to mark procedure as internal since the object already is. Hence this is redundant.

Reply:
Personally, I prefer this way. So, folks don't have to scroll up to the top of the file to figure out if they defined a `public` or `internal` produce.